Resources/Textures/DeltaV/Objects/Storage/barrel.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation PR https://github.com/tgstation/tgstation/pull/38977, modified by rosieposieeee",
```

Comment:
this pr was merged, why not get a permalink of the dmi on the repo

Reply:
👍